updates/work-in-progress/220118_Note.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<update xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.ofw.fi/xml/2011/java-xmldb-updater/UpdaterSchema.xsd">
+  
+  <sql>create table Note (id bigint not null auto_increment, archived bit not null, created datetime not null, creator varchar(255) not null, description longtext, lastModified datetime, lastModifier varchar(255), owner varchar(255) not null, pinned bit not null, priority varchar(255) not null, title varchar(255) not null, type varchar(255) not null, primary key (id));</sql>
```

Comment:
The first sql could be...

`drop table if exists Note;`

...so that when the feature goes live, everyone can upgrade to the proper database version even if they already have the Note table created themselves.